frontend/src/routes/dashboard/connections/new/[type]/+page.svelte

```diff
@@ -1 +0,0 @@
-<h2>New Connection</h2>
```

Comment:
Please stick with the original implementation that uses this route instead of tabs for destination types. (I have a UI design that depends on these being different pages, which is a later iteration.)

Reply:
Ah, ok. Well it was fun to implement the lazy loading. I'll get on this as soon as possible.

Reply:
For context, this is what the final result will look like. The goal is to get the basic functionality working with minimal styling, and then go in and do the pure design work.

![ui](https://github.com/scratchdata/scratchdata/assets/620236/63407dfc-be2a-4c54-b637-25cc4f4aa07b)

Reply:
I have very quickly restored the `[type]` route parameter. Definitely needs a bit more polish.

Before I continue, I want to be sure this is the desired behavior. What I am seeing is a separation of the menu and the form? That means an incorrect click will need to circle back around to the connections page in order to re-open the menu. It also means that the route can be tampered with by the user.

I'm all for implementing what you want. I just want to be sure that's what you _actually_ want?

Reply:
Fair enough.